src/wp-admin/css/login.css

```diff
@@ -44,20 +42,20 @@ p {
 .login .message,
 .login .success,
 .login #login_error {
-	border-left: 4px solid #72aee6;
+	border-left: 4px solid var(--wp-admin--notice-info--border);
 	padding: 12px;
 	margin-left: 0;
 	margin-bottom: 20px;
-	background-color: #fff;
-	box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.1);
+	background-color: var(--wp-admin--surface--background);
+	box-shadow: 0 1px 1px 0 rgb(var(--wp-admin--shadow) / 0.1);
```

Comment:
Does `rgb(var(--wp-admin--shadow) / 0.1)`  work?

Reply:
Yes, this is the space-separated format which was introduced with CSS Colors level 4 - it ends up resolving to `rgb(0 0 0 / 0.1)`, equivalent to `rgba(0, 0, 0, 0.1);`. I wouldn't pay too much attention to the format here, how to handle these transparent values is also an open discussion.

Reply:
Ah, I haven't kept up with that one. I see it's at 92.9% at caniuse.
It looks crazy. I think you have to consider the format, since this would only work with some values. If the properties need to be interchangeable, the format _is_ important.